UI/src/assetTypeConfiguration.js

```diff
@@ -125,7 +125,7 @@
         form: new DynamicAssetForm({
           fields: [
             {label: "massarajoitus", type: 'integer', publicId: "weight", unit: "Kg", required: true, weight: 1},
-            {label: "vihjetieto", type: 'checkbox', publicId: "suggest_box", weight: 2, showAndHide: showSuggestBox, isUnSet: isSuggestBoxUnset}
+            {label: "vihjetieto", type: 'checkbox', publicId: "suggest_box", defaultValue: "0", values: [{id: 0, label: 'Tarkistettu'}, {id: 1, label: 'Vihjetieto'}], weight: 2, showAndHide: showSuggestBox, isUnSet: isSuggestBoxUnset}
```

Comment:
replace id for value here and in similars
```javascript
values: [{value: 0, label: 'Tarkistettu'}, {value: 1, label: 'Vihjetieto'}]

Reply:
Changing this would imply that I would have to change all the fields for all the other assets in order to access them by the value key instead of the id key, not only the suggest_box